demos/sensu-kafka/instructions.md

```diff
@@ -0,0 +1,70 @@
+# Ansible Events + sensu + kafka DEMO
```

Comment:
@Alex-Izquierdo Can you put a picture of all the different components

Reply:
@Alex-Izquierdo Any help for the developers who have minikube?

Reply:
@mkanoor added diagram.

Reply:
@mkanoor I'm not sure if providing multiple environments is in the scope of this task or if it's the priority. The current approach is based on a widely adopted technology that can work straightforwardly in the main operative systems covering the vast majority of local environments. 

In fact, you can [install docker-compose as a standalone binary](https://docs.docker.com/compose/install/compose-plugin/#install-the-plugin-manually) in your minikube VM and execute the demo there. 
```
sudo curl -SL https://github.com/docker/compose/releases/download/v2.7.0/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
```